File: backend/Entities/Schedule.cs
namespace Cinema.Entities;

public class Schedule
{
    public string ScheduleId { get; set; }

    public string RoomId { get; set; }

    public string FilmId { get; set; }

    public DateTime StartTime { get; set; }

    public DateTime EndTime { get; set; }
}

File: frontend/src/types/Schedule.ts
export default interface Schedule {
    scheduleId: string,
    roomId: string,         // Tham chiếu phòng sử dụng của lần chiếu này
    filmId: string,         // Tham chiếu bộ phim được chiếu
    startTime: Date,        // Thời gian bắt đầu chiếu
    endTime: Date           // Thời gian kết thúc của lần chiếu này
}
